test/commands/scanner/run.test.ts

```diff
@@ -21,20 +21,18 @@ describe('scanner:run', function () {
 				const violations = xml.split('<violation');
 				// The first list item is going to be the header, so we need to pull that off.
 				violations.shift();
-				// There should be four violations.
-				expect(violations.length).to.equal(4, `Should be four violations detected in the file:\n ${xml}`);
+				// There should be two violations.
```

Comment:
We should start removing duplicative comments. This is captured in the expect message also.